x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/utils/transforms.ts

```diff
@@ -116,8 +117,15 @@ export const reinstallTransform = async ({
   });
 };
 
-export const getLatestTransformId = (namespace: string): string =>
-  `risk_score_latest_transform_${namespace}`;
+export const getLatestTransformId = (namespace: string): string => {
+  const maxNamespaceLength = 36;
```

Comment:
I think this should be maxTransformIdLength = 64, we care about the max total transform ID length not the max namespace length, currently your code compresses the space id when prefix + namespace length is greater than 36.